auditlog/management/commands/auditlogflush.py

```diff
@@ -83,7 +83,7 @@ def handle(self, *args, **options):
 
 
 class TruncateQuery:
-    SUPPORTED_VENDORS = ("postgresql", "mysql", "sqlite", "oracle", "microsoft")
+    SUPPORTED_VENDORS = ("postgresql", "mysql", "oracle", "microsoft")
```

Comment:
why did you remove sqlite here?

Reply:
SQLite doesn’t support the `TRUNCATE` query ([docs](https://www.sqlite.org/lang_keywords.html)).  
I noticed this while running the SQLite tests.  
From what I found, `DELETE` basically works as the replacement.  

ref: https://www.sqlite.org/lang_delete.html @hramezani